.jenkins/caffe2/build.sh

```diff
@@ -171,7 +171,6 @@ if [[ $BUILD_ENVIRONMENT == *cuda* ]]; then
   export PATH="/usr/local/cuda/bin:$PATH"
 fi
 if [[ $BUILD_ENVIRONMENT == *rocm* ]]; then
-  build_args+=("USE_ROCM=ON")
```

Comment:
Is this an environment variable to be passed to setup.py? All environment variables that start with `USE_` are passed down to CMake from here:

https://github.com/pytorch/pytorch/blob/5e5cbceeba0f936247cd80558e8fba4d8feed00a/tools/setup_helpers/cmake.py#L255

My guess is that the CI passed because `USE_ROCM` defaults to ON if ROCm is detected.

Reply:
All `build_args` are exported as environment variables here https://github.com/pytorch/pytorch/blob/b52a1b99491e8182000475da29214b39af377005/.jenkins/caffe2/build.sh#L255-L257

Reply:
Then if this is for ROCm build, I recommend not removing this line, otherwise it would depend on whether ROCm is detected in the system...

Reply:
Oops sorry I landed before seeing your comment. Could you elaborate why is it not good?
This is CI script. Setting USE_ROCM=ON does not enforce cmake side to error out if ROCM is not properly installed on the CI machines. So effectively this line is useless :-)
For local build, users can still disable ROCm build via `USE_ROCM=OFF`.

Reply:
Yes, I agree, it's effectively useless. Just thought this line offers more clarity :)